app/presenters/mlh_event_presenter.rb

```diff
@@ -20,6 +20,10 @@ def date
     nil
   end
 
+  def time_zone
+    @event.dig('attributes', 'timeZone').gsub(/_/, ' ')
+  end
```

Comment:
```suggestion
  def time_zone
    tz = @event.dig('attributes', 'timeZone')
    time = Time.zone.now.in_time_zone(tz)

    if time.zone.match?(/^\w/)
      time.zone
    else
      time.formatted_offset
    end
  rescue StandardError
    @event.dig('attributes', 'timeZone').gsub(/_/, ' ')
  end
```
Thoughts on doing something like this, so we can show abbreviations or offsets instead of the IANA timezones? I think it might be less confusing for folks with this?

Reply:
Let me ask in the ticket what they would prefer!

Reply:
Preference for IANA in ticket.

Reply:
Sweet, nice and simple.